manualTests/src/main/x/TestSimple.x

```diff
@@ -1,34 +1,10 @@
-@TerminalApp("Simple command tool test")
+// NOTE: THE COMPILATION OF THIS MODULE SHOULD PRODUCE TWO COMPILER ERRORS
 module TestSimple {
-    package cli import cli.xtclang.org;
-
```

Comment:
In the interest of building a corpus with large amounts of XTC code, runnable or negative in that it should not run or compile, can we hang on to old versions of TestSimple somewhere? Even in another repository if you don't want them in the XDK. I read as much XTC code as I can find, so I feel it would be nice to not lose whatever thing was tested in older versions of TestSimple. If they need cleanup, I still think they could be kept somewhere in some corpus repo under a directory from which they are moved to positive or negative examples after we have cleaned them up. Having this incremental effort in place costs us very little, but we do collect more and more XTC code for a corpus that will likely be very useful.  Mostly for people trying to learn XTC, but also for things like training GitHub CoPilot.

Reply:
Well... they are preserved in the git history, aren't they?

Reply:
I use TestSimple as a "code" part of the issue description/explanation/reminder which is also not available in the repository